acceptance/steps/pathway.test.js

```diff
@@ -23,43 +23,7 @@ JestCucumber.defineFeature(feature, test => {
             expect(actual).toEqual(expected)
         });
     });
-
-    test('Pathway page renders controls', ({
-        given,
-        when,
-        then
-    }) => {
-        given('I am a Cyber Capable Person', () => {});
-
-        when('I visit the PSN pathway page', async () => {
-            await driver.visitPage('assessment/psn')
-        });
-
-        then(/^I see a control with id and title \"(.*)\"/, async (expected) => {
-            const firstControlTitle = await driver.findElement('h2.app-task-list__section');
-            const actual = await firstControlTitle.getText();
-            expect(actual).toEqual(expected);
-        });
-    });
     
-    test('Pathway page renders subcontrols', ({
-        given,
-        when,
-        then
-    }) => {
-        given('I am a Cyber Capable Person', () => {});
-        
-        when('I visit the PSN pathway page', async () => {
-            await driver.visitPage('assessment/psn')
-        });
-        
-        then(/^I see a SubControl with the title \"(.*)\"/, async (expected) => {
-            const firstSubControlTitle = await driver.findElement('.app-task-list__task-name');
-            const actual = await firstSubControlTitle.getText();
-            expect(actual).toEqual(expected);
-        });
-    });
-
```

Comment:
see comment on feature files; I don't think these tests should be removed on this branch

Reply:
Just committed some changes that add the pathway test back in. Awaiting Travis build outcome

Reply:
tests passing in [build #782](https://travis-ci.com/github/WeAreSnook/cyber-health/builds/227741631)